docs/tutorials/intro_tutorial.ipynb

```diff
@@ -710,9 +890,30 @@
8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSiQ8AIJX4AABSTa73AADwu67e8ESUThfqPUbDKk6qxEPX13cGj3wAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKnEBwCQSnwAAKlqFh+bN2+OD3/4wzF16tRYvHhx/PznP6/VXQEAE0hN4uPHP/5xdHd3x/333x/PPfdcLFy4MFauXBmvv/56Le4OAJhAahIf3/nOd+LLX/5y3HHHHfHJT34yvv/970dLS0v8y7/8Sy3uDgCYQCZX+4Jvv/127N+/P9avXz9yrKmpKZYtWxZ79uwZc36pVIpSqTRyu7+/PyIifvvb30a5XK7qbOVyOU6dOhWTy01xeqhQ1WtzxuShSpw6NWTPNWbPOew5j13nGN7zm2++Gc3NzVW77okTJyIiolKpvP8MVbvX/+/YsWNx+vTpmD179qjjs2fPjl/+8pdjzt+4cWP09PSMOT5//vxqj0ai2+o9wEXCnnPYcx67zlHLPZ84cSKmTZv2nudUPT7Ga/369dHd3T1ye2hoKH7729/GjBkzolCobvkODAxEZ2dnvPLKK9HW1lbVa3OGPeew5xz2nMeuc9Rqz5VKJU6cOBEdHR3ve27V42PmzJkxadKkOHr06KjjR48ejfb29jHnF4vFKBaLo45ddtll1R5rlLa2Nt/YCew5hz3nsOc8dp2jFnt+v0c8hlX9BadTpkyJa6+9Nnbu3DlybGhoKHbu3BlLliyp9t0BABNMTZ526e7ujnXr1sWiRYvi+uuvj02bNsXg4GDccccdtbg7AGACqUl8fOELX4g33ngj7rvvvjhy5Ehcc801sWPHjjEvQs1WLBbj/vvvH/M0D9VlzznsOYc957HrHB+EPRcq5/JvYgAAqsR7uwAAqcQHAJBKfAAAqcQHAJDqoomPzZs3x4c//OGYOnVqLF68OH7+85/Xe6SGs3v37li9enV0dHREoVCI7du313ukhrRx48a47rrrorW1NWbNmhVr1qyJF198sd5jNZwtW7bEggULRn4R05IlS+Lxxx+v91gN79vf/nYUCoW455576j1KQ9mwYUMUCoVRH1dddVXd5rko4uPHP/5xdHd3x/333x/PPfdcLFy4MFauXBmvv/56vUdrKIODg7Fw4cLYvHlzvUdpaLt27Yqurq7Yu3dv9PX1RblcjhUrVsTg4GC9R2soc+fOjW9/+9uxf//+2LdvX9x8883x+c9/Pv73f/+33qM1rGeffTZ+8IMfxIIFC+o9SkP61Kc+FYcPHx75eOaZZ+o2y0XxT20XL14c1113XXz3u9+NiHd+42pnZ2fcfffd8Td/8zd1nq4xFQqF2LZtW6xZs6beozS8N954I2bNmhW7du2KG2+8sd7jNLTp06fH3/3d38Wdd95Z71EazsmTJ+PTn/50fO9734sHH3wwrrnmmti0aVO9x2oYGzZsiO3bt8eBAwfqPUpEXASPfLz99tuxf//+WLZs2cixpqamWLZsWezZs6eOk0F19Pf3R8Q7PxipjdOnT8cjjzwSg4OD3iaiRrq6uuJP//RPR/1dTXW99NJL0dHRER/5yEfii1/8Yvz617+u2yx1f1fbWjt27FicPn16zG9XnT17dvzyl7+s01RQHUNDQ3HPPffEZz7zmbj66qvrPU7D+cUvfhFLliyJt956Ky699NLYtm1bfPKTn6z3WA3nkUceieeeey6effbZeo/SsBYvXhxbt26NK6+8Mg4fPhw9PT3xx3/8x/HCCy9Ea2tr+jwNHx/QyLq6uuKFF16o63O3jezKK6+MAwcORH9/f/zXf/1XrFu3Lnbt2iVAquiVV16Jr371q9HX1xdTp06t9zgNa9WqVSP/e8GCBbF48eK44oor4j/+4z/q8jRiw8fHzJkzY9KkSXH06NFRx48ePRrt7e11mgou3F133RWPPfZY7N69O+bOnVvvcRrSlClT4mMf+1hERFx77bXx7LPPxj/8wz/ED37wgzpP1jj2798fr7/+enz6058eOXb69OnYvXt3fPe7341SqRSTJk2q44SN6bLLLouPf/zjcfDgwbrcf8O/5mPKlClx7bXXxs6dO0eODQ0Nxc6dOz13y4RUqVTirrvuim3btsVPf/rTmD9/fr1HumgMDQ1FqVSq9xgN5ZZbbolf/OIXceDAgZGPRYsWxRe/+MU4cOCA8KiRkydPxqFDh2LOnDl1uf+Gf+QjIqK7uzvWrVsXixYtiuuvvz42bdoUg4ODcccdd9R7tIZy8uTJURX98ssvx4EDB2L69Okxb968Ok7WWLq6uqK3tzceffTRaG1tjSNHjkRExLRp0+KSSy6p83SNY/369bFq1aqYN29enDhxInp7e+Ppp5+OJ554ot6jNZTW1tYxr1f60Ic+FDNmzPA6pir6+te/HqtXr44rrrgiXnvttbj//vtj0qRJceutt9ZlnosiPr7whS/EG2+8Effdd18cOXIkrrnmmtixY8eYF6FyYfbt2xc33XTTyO3u7u6IiFi3bl1s3bq1TlM1ni1btkRExNKlS0cdf/jhh+P222/PH6hBvf766/GlL30pDh8+HNOmTYsFCxbEE088EcuXL6/3aDBur776atx6663x5ptvxuWXXx433HBD7N27Ny6//PK6zHNR/J4PAOCDo+Ff8wEAfLCIDwAglfgAAFKJDwAglfgAAFKJDwAglfgAAFKJDwAglfgAAFKJDwAglfgAAFKJDwAg1f8DvnKetViEmGQAAAAASUVORK5CYII=",
```

Comment:
This is the line with offending code spell. I think we should ignore and merge... this is not intended for codespell.